challenge-23/challenge-23.js

```diff
@@ -23,3 +23,87 @@ multiplicação (x), então no input deve aparecer "1+2x".
 input;
 - Ao pressionar o botão "CE", o input deve ficar zerado.
 */
+(function(win, doc) {
+  'use strict';
+  var $inputNumber = doc.querySelector( '[data-js="input"]' );
+  var $numberBtn = doc.querySelectorAll( '[data-js="btnNumber"]' );
+  var $operatorBtn = doc.querySelectorAll( '[data-js="operatorBtn"]' );
+  var $equalBtn = doc.querySelector( '[data-js="equalBtn"]' );
+  var $ceBtn = doc.querySelector( '[data-js="ceBtn"]' );
+
+  Array.prototype.forEach.call( $numberBtn, function( button ) {
+    button.addEventListener( 'click', selectedNumber, false );
+  })
+
+  Array.prototype.forEach.call( $operatorBtn, function( button ) {
+    button.addEventListener( 'click', selectedOperator, false );
+  })
+
+  $ceBtn.addEventListener( 'click', selectedCeBtn, false );
+
+  $equalBtn.addEventListener( 'click', selectedEqual, false );
+
+  function selectedNumber() {
+    isInputStarZero( this );
+  }
+
+  function selectedOperator() {
+    removeLastOperator();
+    $inputNumber.value += this.value;
+  }
+
+  function selectedCeBtn() {
+    $inputNumber.value = 0;
+  }
+
+  function selectedEqual() {
+    removeLastOperator();
+    convertForNumber();
+
+  }
+
+  function isInputStarZero(button) {
```

Comment:
-1
Só precisa ajustar o nome dessa função. Acho que você quis dizer `isInputStartsWithZero`, certo?

Lembre que, sempre que você começa uma função com `is`, ela vai retornar um booleano (true ou false), pois você está fazendo uma pergunta: **O input está começando com zero?**

O nome, nesse caso, poderia ser também `startsInputWithZero`. Mas ele deve retornar sempre tru ou false.

Nessa função você está fazendo uma atribuição de valor do `$inputNumber`. Essa atribuição deveria ser feita na função que está chamando essa, dependendo do retorno ser `true` ou `false`.

Conseguiu compreender a ideia? Dessa forma você nomeia a função para que você saiba exatamente o que ela vai retornar, mesmo sem conhecer o que está dentro dela ;)